src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -28,73 +28,113 @@
 import java.util.stream.Collectors;
 
 /**
- * Determines the baseline contig ploidy for germline samples given counts data.
- * These should be either HDF5 or TSV count files generated by {@link CollectFragmentCounts}.
+ * Determines the integer ploidy state of all contigs for germline samples given counts data. These should be either
+ * HDF5 or TSV count files generated by {@link CollectFragmentCounts}.
  *
- * <p>
- *     <b>Important Remark:</b> The Bayesian model underlying this tool assumes integer ploidy states (in contrast
- *     to fractional/variable ploidy states). Therefore, the tool is to be used strictly for germline samples and
- *     for the purpose of sex genotyping and detecting germline aneuploidy in autosomal contigs. The presence of
- *     significant somatic events and mosaicism (e.g. sex chromosome loss and somatic trisomy)
- *     will naturally lead to unreliable results. We strongly recommended inspecting genotyping qualities (GQ)
- *     from the tool output and considering to drop low-GQ contigs in the downstream analyses. Finally, given
- *     the Bayesian status of this tool, we suggest including as many high-quality germline samples as possible
- *     when building a ploidy-model in the COHORT mode (see below). This will downplay the role of questionable
- *     samples and will yield a more reliable estimation of genuine sequencing biases.
- * </p>
+ * <h3>Introduction</h3>
  *
- * <p>This tool has two modes as described below:</p>
+ * <p>Germline karyotyping is a frequently performed task in bioinformatics pipelines, an immediate application of which
+ * is sex determination, as well as informing about expected SNP allele frequencies and copy-ratio plateaus. This
```

Comment:
A bit of a run on.  Also, "frequencies" is typically used in the context of a population and "plateaus" is a bit vague.  Are you trying to say that the karyotype informs the expected distribution of allele-fraction and copy-ratio states for copy-number events?

Reply:
Shortened.